pkgs/development/embedded/srec/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, clangStdenv, fetchFromGitHub }:
+clangStdenv.mkDerivation rec {
+  name = "srec-${version}";
+  version = "2015-03-06";
```

```suggestion
let
  name = "srec";
  version = "0-unstable-2015-03-06";
in
clangStdenv.mkDerivation {
  inherit pname version;
```
Using `rec` is discouraged, please take look at: https://nix.dev/guides/best-practices#recursive-attribute-set-rec

Also:
https://github.com/NixOS/nixpkgs/blob/3e280884c0b0e8222ec6b05a99db01505964e1c3/pkgs/README.md?plain=1#L357